ThunderCloud/ContentController.swift

```diff
@@ -605,6 +617,7 @@ public class ContentController: NSObject {
 			do {
 				
 				// Remove unzip files
+                os_log("Cleaning up `data.tar.gz` and `data.tar` files", log: self.contentControllerLog, type: .debug)
```

Comment:
Guessing that this is caused by indentation type conflicts :)

Reply:
yup